backend/vouch/trustalgo.py

```diff
@@ -0,0 +1,76 @@
+import numpy as np
+from vouch.models import vouch
+
+from core.models import user
+
+
+a = 0.5
+threshold = 0.7
+delta = 10
+eps = 0.000001 
+
+def normalize_trust_values(C):
+    """
+    Normalize the trust values per user, and set diagonal values at 1
+    This makes C an ergotic stochastic matrix
+    """
+    assert(len(C[0])==len(C))
+    for i in range(len(C)):
+        C[i][i] = 1
+        tmp = np.sum(C[i])
+        for j in range(len(C)):
+            C[i][j] = C[i][j] / float(tmp)
+    return C
```

Comment:
In most cases when handling numpy objects, it's preferable to use methods provided by numpy. They are implemented in optimized C code and are significantly more performant than python `for` loops.

Here we could write something like
```py
np.fill_diagonal(C, 1)
return C / C.sum(axis=1, keepdims=True)